toolkits/global/packages/global-meta/scss/10-settings/_nature.scss

```diff
@@ -1,14 +1,17 @@
 /**
  * Meta
- * Springer branding for component
+ * Nature branding for component
  */
 
 // Default button setting
 @import 'default';
 
-$meta--font-size: 1.3rem;
+$meta--font-size: 1.2rem;
 $meta--font-family: $font-family-sans;
-$meta--border: 1px solid greyscale(80);
-$meta--color: greyscale(40);
-$meta--margin-bottom: 4px;
+$meta--border: 1px solid #bbb;
+$meta--color: inherit;
+$meta--line-height: 1.4;
+$meta--margin-bottom: spacing(4);
+$meta--margin-right: spacing(0);
+$meta--padding-right: spacing(4);
```

Comment:
So Nature meta will _always_ want this padding to the right?

Reply:
Yeah and the component wont apply this to the last child.

Unless you mean we have a situation ive missed where the meta is right aligned?

Reply:
This is spacing between meta things? Why padding and not margin?

Reply:
Oh I see. It's for the spacing of border that separates between items